docs/UserGuide.md

```diff
@@ -131,22 +131,24 @@ Examples:
 
 ### Locating contacts by name : `find`
 
-Finds contacts whose names contain any of the given keywords.
+Finds contacts whose details matches all given fields.
 
-Format: `find KEYWORD [MORE_KEYWORDS]`
+Format: `find [n/NAME] [th/TELEGRAM_HANDLE] [e/EMAIL] [a/ADDRESS] [r/ROLE]…​ [nn/NICKNAME]`
```

Comment:
```suggestion
Format: `find [n/NAME] [th/TELEGRAM_HANDLE] [e/EMAIL] [ss/STUDENT_STATUS] [r/ROLE]…​ [nn/NICKNAME]`
```

Reply:
Should we clearly state (in complete sentence) that "the use of multiple keywords is not accepted except for roles"? Giving some examples as well to show what is valid/invalid? Examples are indeed given at the bottom but it will also be better to put it right beside/below the meaning we are trying to convey because people likely wants to understand first thing they finish reading. The example in the examples section can remain as explanation on what the final results were explained but here In this case, I suggest we put examples to show validity, i.e.

Example:
`find n/ Timmy` --> valid
`find n/ Timmy n/ Johnny` --> invalid
`find r/ Admin r/ Marketing` --> valid